docs/Reference/Routes.md

```diff
@@ -227,8 +227,23 @@ checked before parametric and wildcard.*
 
 ```js
 // parametric
-fastify.get('/example/:userId', (request, reply) => {})
-fastify.get('/example/:userId/:secretToken', (request, reply) => {})
+fastify.get('/example/:userId', (request, reply) => {
```

Comment:
I realize they were anonymous arrow functions before your change, but please make all of these handler function full function declarations, e.g.:

```js
function (request, reply) {}
```

Reply:
done this